### PR TITLE
zh-translation:content/zh/about/contribute/github/index.md

### PR DESCRIPTION
**Pull request by tanjunchen**
ref:https://github.com/servicemesher/istio-official-translation/issues/1067

### CODE REVIEW

File: content/zh/about/contribute/github/index.md [hunk 1]
```diff
@@ -12,78 +12,56 @@ aliases:
 keywords: [contribute,community,github,pr]
 ---
 
-We're excited that you're interested in contributing to improve and expand
-our docs! Please take a few moments to get familiar with our procedures before
-you get started.
+我们很高兴您对改进和扩展 Istio 文档感兴趣！ 在开始之前，请花一些时间来熟悉改进与拓展 Istio 文档的流程。
```
Comment (yuxiaobo96) [inline content/zh/about/contribute/github/index.md]:
```suggestion
我们很高兴您对改进和扩展 Istio 文档感兴趣！在开始之前，请花一些时间来熟悉改进与拓展 Istio 文档的流程。
```

File: content/zh/about/contribute/github/index.md [hunk 2]
```diff
@@ -12,78 +12,56 @@ aliases:
...
 
-* If you want to edit an existing page, you can open up the page in your
-  browser and select the **Edit This Page on GitHub** option from the gear menu
-  at the top right of each page. This takes you to GitHub to edit and
-  submit the changes.
+* 如果您想要编辑现有页面，可以在浏览器中打开页面，然后从该页面右上方的齿轮菜单中选择**在 GitHub 上编辑此页面**选项，这将带您到 GitHub 页面进行编辑操作并提交相应的更改。
 
-* If you want to work on the site in general, follow the steps in our
-  [How to add content section](#add).
+* 如果您想要在普通站点上工作，请遵循我们的[如何添加内容](#add)中的步骤。
```
Comment (yuxiaobo96) [inline content/zh/about/contribute/github/index.md]:
```suggestion
* 如果您想使用通用的方式在站点上工作，请遵循我们的[如何添加内容](#add)中的步骤。
```

File: content/zh/about/contribute/github/index.md [hunk 3]
```diff
@@ -12,78 +12,56 @@ aliases:
...
 
-* If you want to review an existing pull request (PR), follow the steps in our
-  [How to review content section](#review)
+* 如果您想要查看现有的拉取请求（PR），遵循我们的[如何查看内容](#review)中的步骤。
```
Comment (yuxiaobo96) [inline content/zh/about/contribute/github/index.md]:
```suggestion
* 如果您想对现有的 pull request（PR）评审，遵循我们的[如何查看内容](#review)中的步骤。
```

File: content/zh/about/contribute/github/index.md [hunk 4]
```diff
@@ -92,37 +70,25 @@ content in detail, follow these steps:
...
+1.  发布评论后，大家经过讨论一致同意合并 PR，请在文本上留下：`/hold cancel`。该命令将取消阻止 PR 合并。
 
-## Previewing your work
+## 预览工作
 
-When you submit a pull request, your PR page on GitHub shows a link to a
-staging site built automatically for your PR. This is useful for you to see
-what the final page looks like to end-users. Folks reviewing your
-pull request also use this staging site to make sure everything looks good.
+当您提交拉取请求（PR）时，您在 GitHub 上的 PR 页面会显示一个指向为您的 PR 自动构建的登入站点的链接，这对于您查看最终用户的最终页面看起来很有用。这个临时的网站，可以确保页面预览看起来正常。
```
Comment (yuxiaobo96) [inline content/zh/about/contribute/github/index.md]:
```suggestion
当您提交 pull request 时，您在 GitHub 上的 PR 页面会显示一个指向为您的 PR 自动构建的登入站点的链接，这对于您查看最终用户的最终页面看起来很有用。这个临时的网站，可以确保页面预览看起来正常。
```

**Comment by tanjunchen** [comment]
@yuxiaobo96  Done

**Pull request merged**